### PR TITLE
Use `gcloud app` instead of `gcloud preview app`

### PR DESCRIPTION
**Pull request by tmatsuo**
We should use `gcloud app` instead of `gcloud preview app` which will go away soon.

### CODE REVIEW

File: php-nginx/README.md [hunk 1]
```diff
@@ -280,7 +280,7 @@ Nginx and php-fpm are communicating via the TCP port 9000.
 You can deploy the app by:
 
 ```sh
-$ gcloud preview app deploy app.yaml
+$ gcloud app deploy app.yaml
```
Comment (bshaffer) [inline php-nginx/README.md]: We should get rid of `app.yaml` here also, since it's no longer necessary

Comment (tmatsuo) [inline php-nginx/README.md]: Done

**Comment by bshaffer** [comment]
LGTM once circleci passes!

**Comment by tmatsuo** [comment]
Yeah #124 needs to be merged for this PR to pass on circleci

**Comment by tmatsuo** [comment]
Rebased and made some tweaks on READMEs.

**Comment by tmatsuo** [comment]
We needed to skip the e2e test when secrets are no available, and it should pass this time!

**Comment by tmatsuo** [comment]
Looking good, merging

**Pull request merged**